apps/qrcode-generator/templates/upload.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+
+{% block main %}
+
+{% if error is not none %}
+<div class="alert alert-danger" role="alert">
+  Something went wrong!
+</div>
+{% else %}
+<div class="alert alert-success" role="alert">
+  QR Code successfully uploaded on the commons
```

Comment:
Rephrase this to: "Your QR code is successfully uploaded to Wikimedia commons!"

Reply:
Done